lib/iStats/settings.rb

```diff
@@ -0,0 +1,37 @@
+module IStats
+  class Settings
+    require 'parseconfig'
+    @configFile="sensors.conf"
+    @configDir=File.expand_path("~/.iStats")+"/"
+
+    class << self
+      def load
+
+        if File.exists?( @configDir+@configFile )
+          $config = ParseConfig.new(@configDir+@configFile)
+
+        else
+          puts "No config file #{@configDir}#{@configFile} found"
+          if !File.exists?(@configDir)
+            Dir.mkdir( @configDir)
+          end
+          file=File.open(@configDir+@configFile,"w+")
+          file.close
+          puts "Running scan"
```

Comment:
Loading the settings shouldn't automatically start a scan. Would rather make this explicit with the `scan` option.